.travis.yml

```diff
@@ -24,7 +24,6 @@ jobs:
         - if [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
             make lint-diff;
           fi
-        - source scripts/run_doctests.sh
```

Comment:
I am not a fan of removing all these tests.  If there are individual tests (like add-book) that we can no longer pass then please add them to the --ignore list in run_doctests.py.  We can also remove the --ignore test_get_ia line which we should now pass.